lib/rollbax/item.ex

```diff
@@ -1,36 +1,70 @@
 defmodule Rollbax.Item do
   def draft(token, envt) do
     {:ok, host} = :inet.gethostname
-    %{"access_token" => token,
-      "data" => %{
-        "server" => %{
-          "host" => List.to_string(host)},
-        "environment" => envt,
-        "language"  => "Elixir",
-        "framework" => "OTP"}}
+    %{access_token: token,
```

Comment:
We should stick with string keys as during JSON encoding they'll be casted to string anyway.
After all we allocate `draft` keys only once.